harness/determined/cli/experiment.py

```diff
@@ -63,7 +63,7 @@ def read_git_metadata(model_def_path: pathlib.Path) -> Tuple[str, str, str, str]
     """
     try:
         from git import Repo
-    except ImportError as e:
+    except ImportError as e:  # pragma: no cover
```

Comment:
We actually don't have a policy around this yet, but I would like to make one.

I propose that a `#pragma: no cover` with no explanation is just as harmful as a line with no coverage in the first place.  If coverage is not needed, then it should be explained when the `#pragma: no cover` is added.

What do you think?

Reply:
I think the pattern is self-explanatory.

Reply:
Ok, that's fair.

There may come a time where somebody thinks something is self-explanatory when others don't think it is... but I suppose we can deal with that problem when/if it arises.

Reply:
I absolutely agree. That's why I didn't mark this as resolved until we reached an agreement. I will mark this as resolved now.